docs/index.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: page
-title: AddressBook Level-3
+title: Intern Buddy
 ---
 
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
The link can be updated with this [link](https://github.com/AY2223S2-CS2103T-T14-3/tp/actions) instead.